src/components/Label/Label.tsx

```diff
@@ -1,13 +1,15 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2020 - 2023 Gemeente Amsterdam
 import { themeColor, themeSpacing } from '@amsterdam/asc-ui'
-import styled, { css } from 'styled-components'
+import styled from 'styled-components'
 
 interface LabelProps {
   /** When true, will render the label with a plain font instead of a bold font */
   inline?: boolean
   /** When false, the Label component will render as an inline-block element without the red header colour */
   isGroupHeader?: boolean
+  /** when true, a label is also black when rendered as group header */
+  isNewDesignSystem?: boolean
```

Comment:
Will this be removed at some time in the future? Because the name (is NewDesignSystem) doesn't clue me in about group headers and that's used for that. Right now I'm thinking this has something to do with the NL design styling..

Reply:
True. I referred to the new design system for now.